code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -232,4 +232,11 @@
 	id = "morphine"
 	results = list("morphine" = 3)
 	required_reagents = list("carbon" = 2, "hydrogen" = 2, "nitrogen" = 1, "oxygen" = 1)
-	required_temp = 480
\ No newline at end of file
+	required_temp = 480
+
+/datum/chemical_reaction/modafinil
+	name = "Modafinil"
+	id = "modafinil"
+	results = list("modafinil" = 5)
+	required_reagents = list("diethylamine" = 1, "ammonia" = 1, "phenol" = 1, "acetone" = 1, "sulphuric acid" = 1)
+	required_catalysts = list("bromine" = 1) // as close to the real world synthesis as possible
```

Comment:
maybe that would be a good recipe standard for chems, but right now this would be one of the most complicated chems while being relatively as powerful as coffee and nicotine

Reply:
Didn't want to make it too easy though ;p this would pretty much be the only currently existing medical chem right now outside of maybe meth that only has a long-lasting positive buff with little to no downside (if used correctly).

Reply:
Nevermind it wasn't actually too hard:
Beaker1:
45 hydrogen
15 nitrogen
15 ethanol
-30 total
Beaker 2:
5 welding fuel
5 carbon
5 hydrogen
-5 total
5 welding fuel
5 oxygen
5 water
5 chlorine
15 Sulphuric acid
1 Bromine
Combine beakers, remove Bromine, done, makes 75u, enough for 10x 7.5u pills (which is only slightly risky). Can also do it via the chemmaster by using the buffer storage to store 1 beaker and also the bromine (which you will need to do anyway because it's a catalyst).